src/ContainerFactory.php

```diff
@@ -34,34 +28,13 @@ public function __invoke() : ContainerInterface
         });
 
         $containerBuilder = new ContainerBuilder;
-        $containerBuilder->addDefinitions(__DIR__ . '/../vendor/php-di/slim-bridge/src/config.php');
         $containerBuilder->addDefinitions($config);
 
-        $cache = new NullAdapter;
         if ($config['config']['enableCache']) {
-            $redisConnection = new Client(['host' => $config['config']['redisHost']]);
```

Comment:
This is moved to config now, this had to be done here because we set a cache on the DI container so kinda had an infinite loop. We needed to create the cache, set it on the container builder, build the container and then set it on the container for other services to use. Now PHP-DI has a compiled container so we move the cache factory back to config.